js/src/admin/components/ExtensionPermissionGrid.js

```diff
@@ -0,0 +1,39 @@
+import PermissionGrid from './PermissionGrid';
+import ItemList from '../../common/utils/ItemList';
+
+export default class ExtensionPermissionGrid extends PermissionGrid {
+  oninit(vnode) {
+    super.oninit(vnode);
+
+    this.extensionId = this.attrs.extensionId;
+  }
+
+  permissionItems() {
+    const permissionCategories = super.permissionItems();
+
+    permissionCategories.items = Object.entries(permissionCategories.items)
+      .filter(([k, v]) => v.content.children.length > 0)
+      .reduce((obj, [key, v]) => {
```

Comment:
I would change the names of these arguments to be more clear about what this code is doing. `k, v` isn't very specific and doesn't give much context.